lib/components/STable.vue

```diff
@@ -338,20 +366,27 @@ function getCell(key: string, index: number) {
   .STable.has-footer & {
     border-radius: 0;
   }
+
+  .block {
+    margin-bottom: -5px;
+    max-height: var(--table-max-height, 65vh);
```

Comment:
This new CSS variable is introduced. It's necessary to limit height in order for virtual-scroller to work.